downstream/modules/eda/con-modifying-simultaneous-activations.adoc

```diff
@@ -3,8 +3,8 @@
 = Modifying the number of simultaneous rulebook activations 
 
 [role="_abstract"]
-By default, {EDAcontroller} allows 12 rulebook activations to run simultaneously.
-If more than 12 rulebook activations are created, the expected behavior is that subsequent rulebook activations wait until there is an available rulebook activation worker.
+By default, {EDAcontroller} allows 12 rulebook activations per node (with two worker or hybrid nodes), resulting in a limit of 24 activations in total, to run simultaneously.
```

Comment:
Maybe this phrase is not correct? I mean, "By default, {EDAcontroller} allows 12 rulebook activations" is correct. The rest is an example with 2 nodes, I'm not sure if its clear, since the number of nodes is chosen by the customer. 

What about:
```suggestion
By default, {EDAcontroller} allows 12 rulebook activations per node. For example, with two worker or hybrid nodes, it results in a limit of 24 activations in total, to run simultaneously.
```

Reply:
This PR does not address the third point of the ticket:
``` A clarification that EDA_MAX_RUNNING_ACTIVATIONS for OCP is a global value since there is no concept of worker nodes in OCP. ```

We need to empathize at some place that max_running_activations behaves differently in OCP and VM's. 
In VM's is what is described, a value per node. In OCP is a global value since in OCP there are no concept of worker nodes. 

VM's:  the limit is the value * number of nodes
OCP: the limit is the value. 

I suggest something like:
```suggestion
By default, {EDAcontroller} allows 12 rulebook activations running at the same time. In case of OCP this is a global limit for the application, in case of installation based on nodes, it is a limit per node. For example, with two worker or hybrid nodes the default limits results in a limit of 24 activations in total. 
```

Reply:
Hi @Alex-Izquierdo. The notes in the jira specified adding the part about OCP in the **Installing on OCP guide** below. So I added it under a discrete heading in the installation scenarios chapter of that guide. The section above is actually in the Performance tuning chapter of the **Using automation decisions guide**.

Reply:
@Alex-Izquierdo - after your comments in the OCP section, I can add a note here in the Using automation decisions guide to alert people who are installing EDA on OCP. Will ping you when the note is complete.